README.md

```diff
@@ -111,42 +113,82 @@ const overviewContent = (
 );
 ```
 
-4. Add integration:
-   In `app-config.yaml` add the integration for gitlab:
-
+4. Add the integration:
+`app-config.yaml` add the integration for gitlab:
 ```yaml
 integrations:
     gitlab:
         - host: gitlab.com
           token: ${GITLAB_TOKEN}
 ```
+**Note:** You can have more than one GitLab instance. 
 
-5. Add proxy config:
+5. Add the GitLab Filler Processor:
```

Comment:
I would briefly explain why we now need a BE plugin too

Reply:
Yes, sure. I want to write it in the migration guide. WDYT?

Reply:
I think you can do both, one for people migrating from old version and just a couple of words in the main readme just to specify why that is needed!

Reply:
ok